### DISCUSSION TITLE
No data from useQuery with enabled: false after reloading the page

### DISCUSSION THREAD

**Discussion by stsiushkevich** [TanStack/query#2609] [Q&A]
Hello! I use an option { enabled: false } for my query to use it with refetch method. When I reload the page the request is completing. Data is returned from the server, but I cannot get it from useQuery. I investigated the code of the library and found such thing: my query doesn't have any observers:

![image](https://user-images.githubusercontent.com/12151060/131103566-eeb44a0d-694b-42ce-b68d-77c7fa0929e6.png)

When I don't reload the page AND use the navigation AND navigate to this page the second time I don't have such problem.

the version is "react-query": "2.26.4"

**Comment by TkDodo** [comment]
please create a codesandbox reproduction, and preferably with v3. If it works with v3 I would suggest upgrading, we have a great guide for that: https://react-query.tanstack.com/guides/migrating-to-react-query-3

**Reply by stsiushkevich** [reply]
I know about v3. But in the current moment I don't have enough time to create a working example on codesandbox.

**Reply by TkDodo** [reply]
Then I hope you understand that I won’t invest time in analyzing this just from the description and a screenshot.
